docs/narr/introduction.rst

```diff
@@ -275,62 +295,63 @@ Here's a few views defined as methods of a class instead:
+in a globally shared location, "the *static* directory". Others use a lookup
+scheme, like an ordered set of template directories. Both of these approaches
+have problems when it comes to customization.
+
+Pyramid takes a different approach. Static assets are located using *asset
+specifications*, strings that contain reference both to a Python package name
+and a file or directory name, e.g. ``MyPackage:static/index.html``. These
+specifications are used for templates, JavaScript and CSS, translation files,
+and any other package-bound static resource. By using asset specifications,
+Pyramid makes it easy to extend your application with other packages without
+worrying about conflicts.
 
-Asset specifications are strings that contain both a Python package name and a
-file or directory name, e.g., ``MyPackage:static/index.html``.  Use of these
-specifications is omnipresent in Pyramid.  An asset specification can refer to
-a template, a translation directory, or any other package-bound static
-resource.  This makes a system built on Pyramid extensible because you don't
-have to rely on globals ("*the* static directory") or lookup schemes ("*the*
-ordered set of template directories") to address your files.  You can move
-files around as necessary, and include other packages that may not share your
-system's templates or static files without encountering conflicts.
-
-Because asset specifications are used heavily in Pyramid, we've also provided a
-way to allow users to override assets.  Say you love a system that someone else
-has created with Pyramid but you just need to change "that one template" to
-make it all better.  No need to fork the application.  Just override the asset
-specification for that template with your own inside a wrapper, and you're good
-to go.
+What happens if another Pyramid package you are using provides an asset you
+need to customize? Maybe that page template needs better HTML, or you want to
+update some CSS. With asset specifications you can override the assets from
+other packages using simple wrappers.
 
 Examples: :ref:`asset_specifications` and :ref:`overriding_assets_section`.
 
-Extensible templating
-~~~~~~~~~~~~~~~~~~~~~
+Use *your* templates
+~~~~~~~~~~~~~~~~~~~~
 
-Pyramid has a structured API that allows for pluggability of "renderers".
-Templating systems such as Mako, Genshi, Chameleon, and Jinja2 can be treated
-as renderers.  Renderer bindings for all of these templating systems already
-exist for use in Pyramid.  But if you'd rather use another, it's not a big
-deal.  Just copy the code from an existing renderer package, and plug in your
-favorite templating system.  You'll then be able to use that templating system
-from within Pyramid just as you'd use one of the "built-in" templating systems.
+In Pyramid, the job of creating a ``Response`` belongs to a :term:`renderer`.
+Any templating system--Mako, Genshi, Chameleon, Jinja2--can be a renderer. In
```

Comment:
I just tried to test Genshi with latest pyramid and python 3.6, turns out Genshi doesn't work with some syntax since 3.4 since ast module changes

https://travis-ci.org/fangpenlin/pyramid_genshi/jobs/235551465#L350

and as Genshi is not updated for a really long while, I don't see too much people using it, even myself is not using it that much, so I am thinking about dropping support of it. You can remove the Genshi support from list now, if there is really something changes, you can add it back later.